modules/tracker/mbt/include/visp3/mbt/vpMbGenericTracker.h

```diff
@@ -309,6 +315,10 @@ class VISP_EXPORT vpMbGenericTracker : public vpMbTracker
   loadModel(const std::map<std::string, std::string> &mapOfModelFiles, bool verbose = false,
             const std::map<std::string, vpHomogeneousMatrix> &mapOfT = std::map<std::string, vpHomogeneousMatrix>());
 
+#ifdef VISP_HAVE_NLOHMANN_JSON
+  void loadJSONSettings(const std::string& settingsFile);
+  void saveJSONSettings(const std::string& settingsFile);
```

Comment:
I suggest to rename to `loadConfigFileJSON()` and `saveConfigFileJSON()`
and introduce `loadConfigFileXML()` that is the actual `loadConfigFile()`

`loadConfigFile()` should be changed to call the appropriate fonction `loadConfigFileJSON()` or `loadConfigFileXML()` depending on the extension